python/dlisio/plumbing/frame.py

```diff
@@ -482,13 +482,16 @@ def fmtstrchannel(self, channel):
     def describe_attr(self, buf, width=80, indent='', exclude=''):
         if len(self.channels) > 0:
             if self.index_type is not None:
+
+                index = self.channels[0]
                 d = OrderedDict()
-                d['Description']      =  self.description
-                d['Indexed by']       =  self.index_type
-                d['Interval']         =  str([self.index_min, self.index_max])
-                d['Direction']        =  self.direction
-                d['Constant spacing'] =  self.spacing
-                d['Index channel']    =  self.channels[0]
+                d['Description']      = self.description
+                d['Indexed by']       = self.index_type
+                d['Units']            = index.units
```

Comment:
1. Not convinced we should add them here at all.
2. Index units?
3. Why self.channels when we have .index method for where first channel isn't actually index one?
4. What will happen if frame is indexed by framenos?
5. Units from index channel are different from units written in the file. Example from our file:

```
   -----
   Frame
   -----
   name   : 800T
   origin : 2
   copy   : 0

   Channel indexing
   --
   Indexed by       : TIME
   Index min        : 33354518 0.5 ms
   Index max        : 35194520 0.5 ms
   Direction        : INCREASING
   Constant spacing : 800 0.5 ms
   Index channel    : Channel(TIME)

   Units from index : ms
```

So units actually are 0.5 ms in the file, whatever that means (half millisecond??) 
So if we print them, we probably need to print it somewhat separately as `35194520 [0.5 ms]`

Reply:
1) Gonna ignore as you don't present any arguments for your claim
2) sure, but seems redundant as it's in a section about indexing
3) Can use that inside the `if`, but the `if`-statement itself does a manual check to avoid the potential logging from `.index`. 
4) no-index is handled exactly like before (`if-else`), no changes to that, and non needed.
5) Good catch, and that would technically be allowed, although stupid. Then I suggest

```
...
Index units : ms
Index min   : 33354518 [0.5 ms]
...
```

Then we do not miss-represent anything. And if the file doesn't make sense, it's not out problem

Reply:
3. I take it back, frame.index return the name of the index channel, not the index channel itself